src/policy/fees.h

```diff
@@ -138,9 +138,9 @@ class CBlockPolicyEstimator
 
     /** Decay of .962 is a half-life of 18 blocks or about 3 hours */
     static constexpr double SHORT_DECAY = .962;
-    /** Decay of .998 is a half-life of 144 blocks or about 1 day */
+    /** Decay of .9952 is a half-life of 144 blocks or about 1 day */
```

Comment:
Remove the number entirely from the comment. No need for a comment that says "x is x"
```suggestion
    /** This decay gives a half-life of 144 blocks or about 1 day */
```
Same for other comments here.

Reply:
I don't think it's a trivial "x is x" comment: it's not trivial to me that a decay of say `.99931` is equivalent to a half-life of 1008 blocks.

Reply:
I agree! That's why I'm saying remove the literal value that's in the code and leave the rest.

Reply:
Ah, ok: will remove if i have to retouch